### ISSUE TITLE
Unable to access WSL2 Ubuntu ports from other LAN connected PCs

### ISSUE THREAD

**Issue by dsjm1** [microsoft/WSL#5798]
# Environment

Windows [Version 10.0.19042.450]
Ubuntu 20.04 LTS
The issue is on WSL 2 
```

# Steps to reproduce
1) Set up ssh on WSL2-Ubuntu-20.04 (I use port 9561)
2) From Win10, use ssh to access the ssh port 9561 using localhost (this works - see grab)
3) Then try ssh to access using the Win10 PC ip addr (10.0.0.19). This fails - see grab).
4) Same problem when trying to access 10.0.0.19:9561 from remote PCs.  A screen grab
   shows attempt to access 10.0.0.19 from 10.0.0.17 (fails) but ssh to 10.0.0.1 (a Ubuntu
   server 18.04 running on an Odroid micro PC works fine.

![in-bound-rules-1](https://user-images.githubusercontent.com/399801/91004103-9a436b00-e616-11ea-86bf-1315607de61d.jpg)
![ipconfig-on10 0 0 19-1](https://user-images.githubusercontent.com/399801/91004104-9b749800-e616-11ea-8805-4e290a8c60df.jpg)
![ssh-from-10 0 0 17-to-10 0 0 19-a](https://user-images.githubusercontent.com/399801/91004105-9c0d2e80-e616-11ea-8269-31551aa8042a.jpg)
![ssh-from-win10-to-wsl2-a](https://user-images.githubusercontent.com/399801/91004106-9ca5c500-e616-11ea-8e70-02b07ee3d5e8.jpg)
![Win10-netstat-1](https://user-images.githubusercontent.com/399801/91004108-9d3e5b80-e616-11ea-8067-a27fcfd9a16a.jpg)

#  Expected behavior
On my Win 10 PC 10.0.0.19, I expected ssh doug@10.0.0.19 -p 9561 to work the same as ssh doug@localhost -p 9561.

<!-- A description of what you're expecting, possibly containing screenshots or reference material. -->

# Actual behavior
See grabs

In essence I am expecting the WSL2 Ubuntu Server to act just like a Virtual Host under Hyper V.
I have other PCs that run Hyper V and I can access them and their ports ok.

**Comment by dsjm1** [comment]
On thinking further about this problem.   As mentioned I can access Hyper-V computers on my LAN.
What it seems is that the WSL2-Ubuntu-20.04 is a virtual machine but while I can access that virtual machine from its host
WSL2 PC running Win 10, I can only do so using 'localhost' as the ip-address.   

In effect, I don't know what hostname or ip address (apart from 'localhost' will allow me access to the Ubuntu-20.04 running under WSL2. Especially from other PCs on the LAN.

The WSL2 Ubuntu-20.04 system can see all my other LAN attached PCs and I can SSH to any of them.

**Comment by dsjm1** [comment]
> Hi! We've identified this issue as a duplicate of another one that already exists in this repository. This specific instance is being closed in favor of tracking the concern over on the referenced thread.
> 
> Thanks for your report!

I just did a further test that showed my problem is due to having more than one Linux distro installed (I had previously had Ubuntu 18.04 & added Ubuntu 20.04 whil upgrading). The moment I added the 20.04 distro, I was no longer able to access either 18.04 or 20.04 remotely other than via 'localhost' on the hosting Win10 PC.

My short term fix is to remove the 18.04 distro.
